acceptance_tests/features/steps/sensitive_data.py

```diff
@@ -0,0 +1,43 @@
+import json
+
+from behave import step
+from tenacity import retry, wait_fixed, stop_after_delay
+
+from acceptance_tests.utilities.database_helper import open_write_cursor
+from acceptance_tests.utilities.rabbit_helper import publish_json_message
+from acceptance_tests.utilities.test_case_helper import test_helper
+from config import Config
+
+
+@step("an UPDATE_SAMPLE_SENSITIVE event is received updating the {sensitive_column}")
+def send_update_sample_sensitive_msg(context, sensitive_column):
+    message = json.dumps(
+        {
+            "event": {
+                "type": "UPDATE_SAMPLE_SENSITIVE",
+                "source": "RH",
+                "channel": "RH",
+                "dateTime": "2021-06-09T13:49:19.716761Z",
+                "transactionId": "92df974c-f03e-4519-8d55-05e9c0ecea43"
+            },
+            "payload": {
+                "updateSampleSensitive": {
+                    "caseId": context.emitted_cases[0]['caseId'],
+                    "sampleSensitive": {sensitive_column: "07898787878"}
+                }
+            }
+        })
+
+    publish_json_message(message, exchange=Config.RABBITMQ_EVENT_EXCHANGE,
+                         routing_key=Config.RABBITMQ_UPDATE_SAMPLE_SENSITIVE_ROUTING_KEY)
+
+
+@retry(wait=wait_fixed(1), stop=stop_after_delay(30))
```

Comment:
I'd rather not have entire steps retried. While in this instance it'll work alright because the step is currently only doing one thing, it's better to specifically retry only code that needs it.

Reply:
seems a bit like gilding the lily to me

Reply:
I suggested it because we already had a helper function pattern for this in the census acceptance tests we could use. If you don't think we're going to need this pattern again in these tests, or add to this step, then fair enough.